roles/openshift_health_checker/test/search_journalctl_test.py

```diff
@@ -0,0 +1,199 @@
+import pytest
+import search_journalctl
+import sys
+
+from time import time
+
+
+SAVED_TIMESTAMPS = list()
+
+if sys.version_info > (3, 0):
+    RE_ERROR_UNEXPECTED_END_REGEXP = "unterminated character set"
+else:
+    RE_ERROR_UNEXPECTED_END_REGEXP = "unexpected end of regular expression"
+
+
+def canned_search_journalctl(get_log_input=None):
+    """Create a Curator check object with canned get_log_input method"""
+    module = search_journalctl  # fails if a module is actually invoked
+    if get_log_input:
+        module._get_log_input = get_log_input
+    return module
+
+
+def get_timestamp_ms(modifier=0, saveto=None):
+    t = str(time() + modifier).split(".")[0] + "000000"
+    if type(saveto) is list:
```

Comment:
Not very Pythonic to be comparing types, but anyway this the preferred way of doing it:

```python
if isinstance(saveto, list):
```

See https://stackoverflow.com/questions/1549801/differences-between-isinstance-and-type-in-python